.github/ISSUE_TEMPLATE/moderator-offboarding.md

```diff
@@ -0,0 +1,13 @@
+---
+name: "Moderator offboarding"
+about: "for moderators who wish to leave"
+labels: "moderator offboarding"
+assignees: r-multiverse/administrators
+---
+
+I would like to leave the R-multiverse [moderator](https://r-multiverse.org/team.html#moderators) team.
+
+## Offboarding process
+
+- [ ] An [administrator](https://r-multiverse.org/team.html#administrators) will remove you from the [R-multiverse GitHub moderator team](https://github.com/orgs/r-multiverse/teams/moderators).
+- [ ] An [administrator](https://r-multiverse.org/team.html#administrators) will remove you [team page on the website](https://github.com/r-multiverse/r-multiverse.github.io/blob/main/team.md).
```

Comment:
transfer to alumni instead.

Reply:
Done as of 55110997f2dc568f8d734a1829dedd95922420ca.